summarization/text_summarization.py

```diff
@@ -51,4 +61,53 @@ def summarize_text(self, text: str, ratio=0.4):
             raise IncorrectInputError(
                 "Ratio should be in the range of [0,1] ")
 
-        return self.text_summarizer(text, ratio)
+        self.text = text
+
+        # get the entities in the text and store it
+        # in a list
+        self._get_entites_from_text()
+        sentence_tokenized_text = sent_tokenize(text)
+
+        text_with_entities_list = list()
+
+        # separate sentences into two:
+        # those that contain entities
+        # and those that don't
+        for sentence in sentence_tokenized_text:
+            sentence_has_entity = False
+
+            for entity in self.entity_list:
+                if entity in sentence:
```

Comment:
Make sure that uppercase/lowercase or spaces or tokenisation difference between the entities and sentences does not effect the comparison here.

Reply:
Updated accordingly.